src/main/scala/tech/christopherdavenport/twitterstorm/Server.scala

```diff
@@ -117,21 +117,30 @@ case class Server[F[_]](tweets: Stream[F, BasicTweet])(
         }
     }
 
+  import Syntax._
+
   def serve(port: Int, ip: String, topN: Int, maxQueueSize: Int, emojiResource: String): Stream[F, Nothing] = {
     val logHeaders = true
     val logBody = true
-    for {
-      scheduler <- Scheduler[F](3)
-      timer <- fs2.async.hold(Duration.Zero, scheduler.awakeEvery(10.millis))
-      counter <- Stream.eval(fs2.async.signalOf[F, BigInt](0))
-      emojiMap <- EmojiParser.emojiMapFromResource(emojiResource)
-      reporter <- tweets.through(StreamTweetReporter(emojiMap, topN, maxQueueSize))
-      nothing <- BlazeBuilder[F] // Stream[F, Nothing] I Would love to remove deadCode Warning Here.
+    (for {
```

Comment:
This is the exciting part.  Your for-comprehension is built in terms of `Resource`.  If any of these components need a shutdown (think: an HTTP client), `Resource.bracket` is available, and we can keep on flatmappin'.